app/controllers/instructor/sections_controller.rb

```diff
@@ -13,7 +13,7 @@ def create
   end
 
   def update
-    current_section.update_attributes(lesson_params)
+    current_section.update_attributes(section_params)
```

Comment:
This is the sections controller, so when we update the params, we should use the `section_params` instead.